Handshake (#5)
RIT-MDRC/catbot_tools: handshake/README.md

# Features

> Arduino sends a "handshake" signal to Raspberry Pi so it knows if the pneumatics system is operating at sufficient pressure. Sets the signal HIGH if the pressure is OK; LOW otherwise.

Reviewer: ```suggestion
Arduino sends a "handshake" signal to Raspberry Pi so Raspberry Pi knows if the pneumatics system is operating at sufficient pressure. Sets the signal HIGH if the pressure is OK; LOW otherwise.
```

> 

Reviewer: ```suggestion
Raspberry Pi program will print "yes" or "no" based on the input received by the Arduino.

```
Add a brief comment on what to expect this program to do.

# Usage

> Connect a jumper wire from the Arduino pin defined in its code (`handshake/arduino/arduino.ino`) to Raspberry Pi's defined pin (`handshake/raspi/receive_handshake.py`). Upload the Arduino code (and set `ENABLE_MOCKING` based on if you have real hardware), and run `handshake/raspi/main.py` to run the sample.

Reviewer: ```suggestion
Connect a jumper wire from the Arduino pin defined in its code (variable OUTPUT_PIN in `handshake/arduino/arduino.ino`) to Raspberry Pi's defined pin (variable INPUT_PIN in `handshake/raspi/receive_handshake.py`). 
Set `ENABLE_MOCKING` based on if you have real hardware in `handshake/arduino/arduino.ino`
Upload the Arduino code
Run `handshake/raspi/main.py` to run the sample.
```

Reviewer: Put one action per line
